js/src/timebomb.ts

```diff
@@ -29,15 +35,23 @@ export function getConfig(options: Partial<Config>): Config {
   };
 }
 
-export function warnAfter(time: Date, options: Partial<Config> = {}) {
+export async function warnAfter(time: Date, options: Partial<Config> = {}) {
```

Comment:
I'd prefer not to clutter up the lib with async versions of things that shouldn't take time -- given that `await undefined === await Promise.resolve(undefined)`, i'm not sure if there's a use case for this.

Reply:
My intention here was to have all async versions return a promise to make this lib easier to use.

For the same reason I included a sync and async version for everything.

Reply:
Yeah, I get the intention, but I think waitAfter (non-sync) would be the only use case I can imagine for using the return value, so if it comes down to a choice between removing a bunch of duplicate functions where it's unclear why they exist vs returning an instantly resolving promise, I'd choose the former.

Or I'd just modify the originals to return Promise.resolve(undefined) and remove the sync versions entirely, but even that I feel is unnecessary. 

Thanks for your input!